### PR TITLE
Documentation: style fix

### PR DESCRIPTION
**Pull request by stevenpost**
Use a code block for the commands so they are displayed properly on
GitHub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2546,14 +2546,18 @@ Read the complete module [contribution guide](https://docs.puppetlabs.com/forge/
 This project contains tests for both [rspec-puppet](http://rspec-puppet.com/) and [beaker-rspec](https://github.com/puppetlabs/beaker-rspec) to verify functionality. For in-depth information please see their respective documentation.
 
 Quickstart:
-  Ruby > 1.8.7
```
Comment (igalic) [inline README.md]: i'd insert an enter here

File: README.md [hunk 2]
```diff
@@ -2546,14 +2546,18 @@ Read the complete module [contribution guide](https://docs.puppetlabs.com/forge/
...
+####Ruby > 1.8.7
```
Comment (igalic) [inline README.md]: and here

**Comment by stevenpost** [comment]
*added some line feeds*

**Comment by igalic** [comment]
can we make travis not trigger on documentation changes…?

**Comment by stevenpost** [comment]
Am I even able to do that on my PR?

**Comment by igalic** [comment]
i don't think so, that's more of a general question…
shouted into the void ;)

**Pull request merged**